template/src/navigation/StackNavigation.tsx

```diff
@@ -12,41 +12,63 @@ export const componentMatch = (stackName: string): Element | string => {
   }
 }
 
-const Stack = createNativeStackNavigator()
+export type HomeStackParamList = {
+  HomeScreen: undefined
+}
+
+export type AuthStackParamList = {
+  LoginScreen: undefined
+  SignUpScreen: undefined
+}
+
+export type AppStackParamList = AuthStackParamList & HomeStackParamList
+
+const Stack = createNativeStackNavigator<AppStackParamList>()
 
-export const HomeNavigator = (): Element => {
+interface IProps {}
+
+export const HomeNavigator = (props: IProps) => {
   return (
     <Stack.Navigator>
       <Stack.Screen
-        name={RouteKey.HomeScreen}
+        name={'HomeScreen'}
```

Comment:
Why are you hard-code this? Please use `RouteKey` for this

Reply:
updated